src/main/resources/static/js/ingredientManagerScreen.js

```diff
@@ -0,0 +1,117 @@
+var _restEndpoint = '/api/ingredients/';
+var _tableElement = $('#ingredientTable');
+var _addButton = $('#addButton');
+var _modalElement = $('#entryModal');
+var _deleteElement = $('#btndelete');
+
+var _dataTable = _tableElement.DataTable({
+    ajax: {
+        url: '/api/ingredients/',
```

Comment:
je hebt al een variabele voor api.ingredients aangemaakt, die zou je eventueel hier gewoon kunnen gebruiken. dat is wat netter